### PR TITLE
Diagnostics analysis in daas

### PR DESCRIPTION
**Pull request by poppastring**
## Summary

Announce a new feature available in Diagnose and Solve for analyzing dumps in the portal

### CODE REVIEW

File: _posts/2022-12-08-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+title: "Improving the dump analysis journey in Diagnose and Solve"
+author_name: "Mark Downie, Puneet Gupta"
+toc: true
+toc_sticky: true
+category: diagnostics
+---
+
+We are happy to announce that our most recent improvements to **Diagnose and Solve** allows you to begin the analysis on a collected memory dump immediately in the Azure portal!
```
Comment (puneetg1983) [inline _posts/2022-12-08-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md:9]: Change allows to allow

File: _posts/2022-12-08-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+For problems that do not manifest in logs or that you cannot investigate by debugging locally you might attempt to capture a diagnostics artifact, like a memory dump, while the issue is active in your production environment. Diagnose and Solve diagnostics tools have enabled you to [capture artifacts like memory dumps on demand or by using **Custom Auto Heal** for both Linux and Windows](https://azure.github.io/AppService/2021/11/01/Diagnostic-Tools-for-ASP-NET-Core-Linux-apps-are-now-publicly-available.html).
```
Comment (puneetg1983) [inline _posts/2022-12-08-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md:11]: Comma after locally

**Review by puneetg1983 — APPROVED** [review]

**Comment by poppastring** [comment]
@JasonFreeberg This is ready to go. Thx

File: _posts/2022-12-05-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+<ItemGroup>
+   <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
+</ItemGroup>
+```
+
+There is also source code link support for [Azure Repos, Azure DevOps, GitLab, Bitbucket, gitweb, and gitea](https://learn.microsoft.com/visualstudio/debugger/how-to-improve-diagnostics-debugging-with-sourcelink#alternative-source-control-providers).
+
+## Summary
+
+Please check out the improved Diagnose and Solve experiences for App Services for Windows and App Services for Linux! We are actively working on Diagnostics Analysis for traces and we are always interested in your feedback.
```
Comment (yunjchoi) [inline _posts/2022-12-05-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md:94]:
```suggestion
Please check out the improved Diagnose and Solve Problem experiences for App Services for Windows and App Services for Linux! We are actively working on Diagnostics Analysis for traces and we are always interested in your feedback.
```

File: _posts/2022-12-08-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
+---
+title: "Improving the dump analysis journey in Diagnose and Solve"
```
Comment (yunjchoi) [inline _posts/2022-12-08-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md]:
```suggestion
title: "Improving the dump analysis journey in Diagnose and Solve Problems"
```
A suggestion to align the name with the name displayed in the portal

File: _posts/2022-12-05-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+title: "Improving the dump analysis journey in Diagnose and Solve"
+author_name: "Mark Downie, Puneet Gupta"
+toc: true
+toc_sticky: true
+category: diagnostics
+---
+
+We are happy to announce that our most recent improvements to **Diagnose and Solve** allow you to begin the analysis on a collected memory dump immediately in the Azure portal!
+
+For problems that do not manifest in logs or that you cannot investigate by debugging locally, you might attempt to capture a diagnostics artifact, like a memory dump, while the issue is active in your production environment. Diagnose and Solve diagnostics tools have enabled you to [capture artifacts like memory dumps on demand or by using **Custom Auto Heal** for both Linux and Windows](https://azure.github.io/AppService/2021/11/01/Diagnostic-Tools-for-ASP-NET-Core-Linux-apps-are-now-publicly-available.html).
```
Comment (yunjchoi) [inline _posts/2022-12-05-Making-diagnostics-analysis-easier-in-Diagnose-and-Solve.md:11]:
```suggestion
For problems that do not manifest in logs or that you cannot investigate by debugging locally, you might attempt to capture a diagnostics artifact, like a memory dump, while the issue is active in your production environment. Diagnostics tools found under Diagnose and Solve Problems have enabled you to [capture artifacts like memory dumps on demand or by using **Custom Auto Heal** for both Linux and Windows](https://azure.github.io/AppService/2021/11/01/Diagnostic-Tools-for-ASP-NET-Core-Linux-apps-are-now-publicly-available.html).
```

**Review by yunjchoi — CHANGES_REQUESTED** [review]
Amazing article! I just requested some minor changes to align the name of our experience with the name displayed in the portal and some grammar improvements I was able to spot. Thank you!

**Comment by poppastring** [comment]
Thanks @yunjchoi I have applied those changes. Could you please approve so that we can complete the merge?

**Review by yunjchoi — APPROVED** [review]
Lgtm!

**Pull request merged**